packages/gestalt/src/Video.test.js

```diff
@@ -136,3 +136,18 @@ test('Video with objectFit', () => {
   ).toJSON();
   expect(tree).toMatchSnapshot();
 });
+
+test('Video with startTime', () => {
```

Comment:
Thanks for adding tests! I'm not sure if a snapshot is very valuable for this test, though. What would be useful is a test that passes a `startTime` and verifies that the `<video>` element seeks to that time.

Reply:
Yeah I can kill this one if thats easier, I noticed some of this testing isn't possible without a headed browser testing suite which is why I moved it here along with some of the other assertions like `objectFit` where they're more like contract tests

Reply:
I might be misunderstanding the timing of the logic, but I was thinking of something like passing a `startTime` and a `ref` and then reading the `currentTime` off the ref to confirm that it matches. I think that should be possible?

Reply:
The attrs and properties of all the nodes are super mocked out under test in headless environments things like `disableremoteplayback` are there but stuff it actually tests against the video asset like times are proxied out

Reply:
Ah cool, no worries. We _just_ switched to all headless (like this morning) and I haven't had a chance to play around with it much, so I'm not familiar with it yet. If some sort of logic test like that seems possible, let's do it; if it looks like it'd just be a snapshot, don't worry about adding the test.

Reply:
I think snapshot is all we can do here since `node.currenttime` is mocked out totally in headless. My guess is any of the attributes that reflect video state vs the broad config ones aren't available in headless since its probably mocking them out